test/Makefile

```
@@ -47,13 +47,19 @@ CFLAGS += -fsanitize=address
 LDFLAGS += -fsanitize=address
 endif
 
+# Support a NO CRYPTO build
+ifeq ($(NOCRYPTO),1)
+CFLAGS += -DWOLFHSM_NO_CRYPTO -Wpadded
+endif
+
 # wolfHSM-specific defines
-CFLAGS += -DWH_CONFIG
+CFLAGS += -DWH_CONFIG -DWOLFHSM_NO_CRYPTO
```

I assume this should only be defined for the NOCRYPTO option right? 

```suggestion
CFLAGS += -DWH_CONFIG
```

---

Gaah!! Fixed.